pom.xml

```
@@ -57,17 +57,15 @@
         </developer>
     </developers>
     <properties>
-        <apache.rat.version>0.12</apache.rat.version>
-        <jacoco.maven.version>0.8.9</jacoco.maven.version>
```

```suggestion
        <jakarta.annotation.version>2.1.1</jakarta.annotation.version>
        <jacoco.maven.version>0.8.9</jacoco.maven.version>
```

@eliasnogueira you forget the jacoco properties.